### PR TITLE
Reviews component skeleton/outline

### PR DESCRIPTION
**Pull request by slhodak**
Adds a skeleton of the reviews component and its subcomponents
Renders some data and styling to the summary section
Fixes precision problem: properly limits decimal-depth for all floats from Faker

This branch should have been named "Reviews Components Outline" or something of that nature, and I would like to merge it now and limit the scope of future branches to smaller pieces of the module.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,4 +68,63 @@ npm install -g webpack-cli
...
\ No newline at end of file
+  - reviews=# select * from reviews;
+
+## API
+
+### Restaurant Impression
+
+#### HTTP request
+
+GET http://127.0.0.1:3010/:id/impression
```
Comment (winniefong1021) [inline README.md]: I get a 500 status error to this route, but the route for /reviews is successful and I was able to see the data come back.

File: config/localRole.js [hunk 2]
```diff
@@ -0,0 +1 @@
+module.exports = 'macuser';
```
Comment (winniefong1021) [inline config/localRole.js]: I saw this file in the gitignore, but it looks like it still got pushed up to github...

Comment (slhodak) [reply config/localRole.js]: Thanks, looking into it

**Comment by winniefong1021** [comment]
Everything else looks good!

File: README.md (hunk 1 above)
Comment (slhodak) [reply README.md]: Did you use a number between 1 and 5 as the restaurant id?

**Pull request merged**